packages/web-components/src/utils/utils.ts

```diff
@@ -21,19 +21,8 @@ export function getSlottedNodes(
   root: HTMLElement,
   nodeName: string | null,
 ): Array<Node> {
-  // If the browser is using the shadowDOM, the childNodes should be an array of two things:
-  // A `<style>` element and a `<slot>` element
-  // Chrome, Firefox, Safari, Edge - literally every modern browser will use shadowDOM.
-  // This is purely for IE compatibility
-  const hasShadowDOM =
-    Array.from(root.shadowRoot.childNodes).filter(
-      (node: any) => node.tagName === 'SLOT',
-    ).length > 0;
-
```

Comment:
This did not account for a slot _not_ being a direct descent of the root `Host` which was the case for the USWDS markup so it was causing issues for that variant.

```html
<Host>
   <fieldset>
      [....]
    <slot></slot>
  </fieldset>
</Host>
```

Ultimately though, this appears to be a relic for IE11 which is no longer supported so I opted to remove it entirely as tech debt.